src/backend/joanie/tests/core/test_api_admin_products.py

```diff
@@ -484,3 +484,21 @@ def test_admin_api_product_reorder_target_course(self):
         self.assertEqual(relations.get(course=courses[0]).position, 2)
         self.assertEqual(relations.get(course=courses[2]).position, 3)
         self.assertEqual(relations.get(course=courses[4]).position, 4)
+
+    def test_admin_api_product_update_empty_instructions(self):
+        """
+        Staff user should be able to update a product with empty instructions.
```

Comment:
Why not testing with empty new line and whitespace ? This is the expected behaviour